File: Practice/Easy/217.ContainsDuplicate/python/solution2.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        """
        T: O(n log n)
        M: O(1)
        :param nums:
        :return:
        """
        nums.sort()
        for i in range(1, len(nums)):
            if nums[i] == nums[i - 1]:
                return True
        return False

File: Practice/Easy/217.ContainsDuplicate/go/solution2.go
// T: O(n log n) M: O(1)
func containsDuplicate(nums []int) bool {
    sort.Ints(nums)
    for i := 1; i < len(nums); i ++ {
        if nums[i] == nums[i - 1] {
            return true

        }
    }
    return false
}